components/constants/webui_url_constants.h

```diff
@@ -97,6 +97,9 @@ inline constexpr char kRewriterUIHost[] = "rewriter";
 inline constexpr char16_t kTransactionSimulationLearnMoreURL[] =
     u"https://github.com/brave/brave-browser/wiki/Transaction-Simulation";
 
+inline constexpr char kNTPSponsoredRichMediaUrl[] =
+    "chrome-untrusted://rich-media/";
```

Comment:
this seems much too generic and should be prefixed with something New Tab related IMO

Reply:
@petemill how about `rich-ntt`?